tests/trainers/test_regression.py

```diff
@@ -198,3 +218,73 @@ def test_predict(self, model_kwargs: dict[Any, Any], fast_dev_run: bool) -> None
             max_epochs=1,
         )
         trainer.predict(model=model, datamodule=datamodule)
+
+    def test_invalid_loss(self, model_kwargs: dict[str, Any]) -> None:
+        model_kwargs["loss"] = "invalid_loss"
+        match = "Loss type 'invalid_loss' is not valid."
+        with pytest.raises(ValueError, match=match):
+            RegressionTask(**model_kwargs)
+
+
+class TestPixelwiseRegressionTask:
+    @pytest.mark.parametrize(
+        "name,classname,batch_size,loss",
+        [
+            ("inria", InriaAerialImageLabelingDataModule, 1, "mse"),
+            ("inria", InriaAerialImageLabelingDataModule, 2, "mae"),
```

Comment:
Testing regression on Inria binary [0, 1] masks for now since we don't have a readily available pixelwise regression datamodule.